.github/workflows/testing.yml

```diff
@@ -38,6 +38,26 @@ jobs:
           command: lint
           config: .github/ct.yaml
 
+  unit-test:
+    name: Unit test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Unit tests
+        run: |
+          go mod vendor
```

Comment:
tests runner should install missing deps automatically, no need to vendor them in advance